lib/zendesk_apps_tools/command.rb

```diff
@@ -73,11 +75,18 @@ def validate
         end
         say_error_and_exit error
       end
+
       valid = errors.none?
 
       if valid
-        app_package.warnings.each { |w| say w.to_s, :yellow }
-        say_status 'validate', 'OK'
+        require 'zendesk_apps_support/app_version'
+        @zas ||= ZendeskAppsSupport::AppVersion.new(manifest.version)
+
+        if @zas.valid_for_update?
+          app_package.warnings.each { |w| say w.to_s, :yellow }
+          say_status 'validate', 'OK'
+        end
+        sunsetted app_package.manifest.framework_version
```

Comment:
Looks like they are the same: https://github.com/zendesk/zendesk_apps_tools/blob/master/lib/zendesk_apps_tools/package_helper.rb#L10